src/test/functional/cypress/integration/feature/admin/jobCategories.spec.js

```diff
@@ -0,0 +1,99 @@
+describe('Admin - Job Category', function () {
+  beforeEach(function () {
+    cy.task('db:reset');
+    cy.viewport(1280, 720);
```

Comment:
```suggestion
    cy.fixture('viewport').then(({HD}) => {
      cy.viewport(HD.width, HD.height);
    });
```
@Super-Chama is this correct?

Reply:
yeah this should be the way